exercises/error-handling/Cargo.toml

```diff
@@ -0,0 +1,6 @@
+[package]
+name = "error-handling"
+version = "1.0.0"
```

Comment:
there is no 1.0.0 in https://github.com/exercism/problem-specifications/tree/master/exercises/error-handling therefore this must be `0.0.0` or `0.1.0` your choice.